pom.xml

```diff
@@ -65,7 +65,7 @@
         <dep.gcs.version>1.9.17</dep.gcs.version>
         <dep.alluxio.version>313</dep.alluxio.version>
         <dep.slf4j.version>1.7.32</dep.slf4j.version>
-        <dep.kafka.version>2.3.1</dep.kafka.version>
+        <dep.kafka.version>3.7.1</dep.kafka.version>
```

Comment:
@adkharat I wanted to understand if there was a specific reason for selecting Kafka version 3.7.1 instead of opting for the latest 3.8.x release?

Reply:
@imjalpreet Reason:-  The vulnerabilities CVE-2022-34917 and CVE-2023-25194 both have been addressed in version 3.7.1. 
I haven’t tested with the 3.8.x release, but I can give it a try if needed.

Reply:
@ZacBlanco Latest Kafka version is 3.9.0. 
Pushed changes for 3.9.0

Reply:
Great! Thank you